src/components/lyne-text-input/lyne-text-input.stories.js

```diff
@@ -0,0 +1,462 @@
+import events from './lyne-text-input.events.ts';
```

Comment:
as mentioned before, since there are no events handled at the moment, i'd remove it from here as well